delivery_postlogistics/views/stock.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2019 Camptocamp SA
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_picking_withcarrier_out_form" model="ir.ui.view">
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="delivery.view_picking_withcarrier_out_form" />
+        <field name="arch" type="xml">
+            <field name="state" position="before">
+                <button
```

Comment:
This action already exists in standard and is called "send to shipper". This calls "postlogistics_send_shipping"

Reply:
fixed, removed this button and the action will be called in `def send_to_shipper()`